lua/lint/linters/golangcilint.lua

```diff
@@ -8,6 +8,7 @@ local severities = {
 return {
   cmd = 'golangci-lint',
   append_fname = false,
+  append_dname = true,
```

Comment:
Could you instead change the args to:

```lua
  args = {
    'run',
    '--out-format',
    'json',
    function()
      return vim.fn.fnamemodify(vim.api.nvim_buf_get_name(0), ":h")
    end
  },
```

And remove the `append_dname` option?
I'd like to refrain from adding more general options until it is required by more than just one linter.

Reply:
https://github.com/mfussenegger/nvim-lint/pull/265/files